frontend/assets/style/components/_custom-markdowns.scss

```diff
@@ -137,6 +137,11 @@
 
   .link {
     border-bottom-color: currentColor;
+    margin-left: 2px;
+
+    &:has(+ *) {
+      margin-right: 2px;
```

Comment:
I'm not sure using `margin` is the right solution here. Using `content: ' ';` would provide a space visually. However, no CSS solution will give a space when copying text, so probably the space should be part of the DOM.